infrastructure/variables.tf

```diff
@@ -180,3 +180,7 @@ variable "appinsights_instrumentation_key" {
   description = "Instrumentation key of the App Insights instance this webapp should use. Module will create own App Insights resource if this is not provided"
   default     = ""
 }
+
+variable "process_payments_enabled" {
+  default = "true"
```

Comment:
may be disable it for now and turn it on when needed ?

Reply:
you mean disable by default and enable on aat and demo?
we have to enable on aat and demo for QA

Reply:
There's always edgy case whether one should default to `true` or `false`